### PR TITLE
[HOPSWORKS-2014]

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

**Comment by ErmiasG** [comment]
logicalclocks/hops-testing-ee/pull/197

File: hops/constants.py [hunk 1]
```diff
@@ -517,6 +517,24 @@ class REST_CONFIG:
...
+    JSON_ACCESS_KEY_ID = "accessKeyId"
+    JSON_SECRET_KEY_ID = "secretAccessKey"
+    JSON_SESSION_TOKEN_ID = "sessionToken"
+    JSON_ARRAY_ITEMS = "items"
+    JSON_CLOUD_ROLE = "cloudRole"
+
+    JSON_KAFKA_TOPIC_SCHEMA_VERSION = "schemaVersion"
+    JSON_KAFKA_TOPIC_NAME = "name"
+    JSON_KAFKA_NUM_PARTITIONS = "numOfPartitions"
+    JSON_KAFKA_NUM_REPLICAS = "numOfReplicas"
```
Comment (tkakantousis) [inline hops/constants.py:536]: wrong copy paste?

File: hops/credentials_provider.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    Returns:
+        A role arn
+    """
+    json_content = _get_roles(role_id=role_id)
+    return json_content[constants.REST_CONFIG.JSON_CLOUD_ROLE]
+
+
+def _set_spark_hadoop_conf(json_content):
+    spark = None
+    if "IS_HOPS_DRIVER" in os.environ:
```
Comment (tkakantousis) [inline hops/credentials_provider.py:83]: Maybe move to constants?

**Review by tkakantousis — CHANGES_REQUESTED** [review]
JIRA title missing from PR

**Review by tkakantousis — APPROVED** [review]

**Pull request merged**